ABCD/data_convert.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+from pathlib import Path
+
+import pandas as pd
+
+
+def get_args():
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter, description=__doc__)
+    parser.add_argument('-t', '--tabular-data', type=Path, action='store', dest='tab_data', metavar='TAB_DATA_DIR',
+                        help='Path to directory with tabular data files in \'.txt\' format.')
+    parser.add_argument('-d', '--dictionaries', type=Path, action='store', dest='dicts', metavar='DICTS_DIR',
+                        default=Path('.'), help="Path to directory with data dictionaries in \'.csv\' format.")
```

Comment:
The README.md needs a section that tells the user how to prepare the data before running this script. This should include (in this case) the user MUST download the data dictionaries and use dictionary.py first before this. Then the dictionaries will be under this `ABCD/` subfolder in a `phenotype/` subfolder.

Reply:
They could run `data_convert.py` before `dictionary.py` but they'll need to download the data from NDA and dictionaries from GitHub/NDA. I'll remove the argparser function.